### ISSUE TITLE
A little confuse about the implement 

### ISSUE THREAD

**Issue by jarvisWang0903** [gabrielhuang/reptile-pytorch#4]
Hi Gabriel,

Thank you for sharing the code!

I have a little confuse about this implement~

In my opinion, meta-learning is use meta-train set for several update and then merge all loss [calculate  on meta-test set ] from different episode and sum over it, and backward the loss on meta network to get the grads and then update the base network use the grads. But in your implement i did't see the meta-test set. It seems that you use the base network's grads minus meta network's grads directly which calculate on the meta-train set.

**Comment by jarvisWang0903** [comment]
It seems like this implement.

https://github.com/dragen1860/Reptile-Pytorch/blob/0bd5d777c167fa1aea66b5dc325a7fc4e56632ab/meta.py#L76


https://github.com/dragen1860/Reptile-Pytorch/blob/0bd5d777c167fa1aea66b5dc325a7fc4e56632ab/meta.py#L210
